ProconXInput.sln

```diff
@@ -0,0 +1,31 @@
+﻿

```

Comment:
This isn't a multi-project repo, so solution files should be excluded.

Reply:
Yes, I know but VS will generate the file no matter what you do (as far as I know). So the sln is the "normal" entypoint.